data/human/kestrel.txt

```diff
@@ -249,23 +249,53 @@ mission "Kestrel Available"
 				decline
 
 
+shipyard "Kestrel"
 
 event "kestrel available: more weapons"
 	set "kestrel available"
-	shipyard "Tarazed Advanced"
+	shipyard "Kestrel"
 		"Kestrel (More Weapons)"
+	planet Tarazed
+		add shipyard "Kestrel"
 event "kestrel available: more engines"
 	set "kestrel available"
-	shipyard "Tarazed Advanced"
+	shipyard "Kestrel"
 		"Kestrel (More Engines)"
+	planet Tarazed
+		add shipyard "Kestrel"
 event "kestrel available: more shields"
 	set "kestrel available"
-	shipyard "Tarazed Advanced"
+	shipyard "Kestrel"
 		"Kestrel (More Shields)"
+	planet Tarazed
+		add shipyard "Kestrel"
 event "kestrel available: more bays"
 	set "kestrel available"
-	shipyard "Tarazed Advanced"
+	shipyard "Kestrel"
 		"Kestrel (More Bays)"
+	planet Tarazed
+		add shipyard "Kestrel"
+
+
+# Remove the Kestrel from Tarazed Advanced for saves that obtained it prior to 0.10.7.
+
+mission "Kestrel Sales Patch'
+	invisible
+	landing
+	to offer
+		has "kestrel available"
+	on offer
+		event "kestrel sales patch"
+		fail
+	
+event "kestrel sales patch"
+	planet Wayfarer
+		add shipyard "Kestrel"
+	shipyard "Tarazed Advanced"
+		remove "Kestrel (More Weapons)"
+		remove "Kestrel (More Enginess)"
+		remove "Kestrel (More Shields)"
+		remove "Kestrel (More Bays)"
```

Comment:
On an older save file, the shipyard "Kestrel" is going to be empty, so this removes it from the shipyard that it was present in and then adds an empty shipyard to Wayfarer, so the Kestrel is now not available.

I'm not sure it's really a huge problem if older save files have the Kestrel available at Zug, but if that is something you want to change, you're going to need four missions and four events: one to add each variant to the "Kestrel" shipyard based on which variant was previously unlcoked.

Reply:
I say just let old saves have the extra Kestrel.